packages/fast-components-react-base/src/image/image.spec.tsx

```diff
@@ -0,0 +1,135 @@
+import * as React from "react";
+import * as ShallowRenderer from "react-test-renderer/shallow";
+import * as Adapter from "enzyme-adapter-react-16";
+import { configure, shallow } from "enzyme";
+import examples from "./examples.data";
+import { generateSnapshots } from "@microsoft/fast-jest-snapshots-react";
+import {
+    IImageClassNameContract,
+    IImageHandledProps,
+    IImageManagedClasses,
+    IImageUnhandledProps,
+    ImageProps,
+} from "./image";
+
+/*
+ * Configure Enzyme
+ */
+configure({adapter: new Adapter()});
+
+describe("image", (): void => {
+    generateSnapshots(examples);
+});
+
+describe("image unit-tests", (): void => {
+    const Component: React.ComponentClass<IImageHandledProps & IImageManagedClasses> = examples.component;
+    const managedClasses: IImageClassNameContract = {
+        image: "image-class",
+        image_round: "image-round",
+        picture: "picture"
+    };
+    const alt: string = "Image alt text test string";
+
+    test("should return an object that includes all valid props which are note enumarated as handledProps", () => {
+        const handledProps: IImageHandledProps & IImageManagedClasses = {
+            managedClasses,
+            alt,
+            src: "https://placehold.it/200x200"
+        };
+        const unhandledProps: IImageUnhandledProps = {
+            "aria-hidden": true
+        };
+        const props: ImageProps = {...handledProps, ...unhandledProps};
+        const rendered: any = shallow(
+            <Component {...props} />
+        );
+
+        expect(rendered.prop("aria-hidden")).not.toBe(undefined);
+        expect(rendered.prop("aria-hidden")).toEqual(true);
+    });
+
+    test("should correctly NOT render anything if `src` and `vp1` props are both undefined", () => {
+        const rendered: any = shallow(
+            <Component managedClasses={managedClasses} alt={alt} />
+        );
+
+        expect(rendered.type()).toEqual(null);
+    });
+
+    test("should correctly render an `<img/>` element if `src` prop is passed", () => {
+        const rendered: any = shallow(
+            <Component managedClasses={managedClasses} alt={alt} src={"https://placehold.it/20x20"} />
+        );
+
+        expect(rendered.type()).toBe("img");
+    });
+
+    test("should correctly render a `<Picture>` element if `vp` props are passed", () => {

```

Comment:
Should this be <picture> element (lowercase)? Should we specify vp1/vp2/vp3/vp4/vp5/vp6 as `vp` props might not mean anything?

Reply:
Element should be lowercase as it refers to HTML.